apps/shell/tash_internal.h

```diff
@@ -91,6 +91,11 @@ extern int tash_sleep(int argc, char **args);
 #ifdef CONFIG_TASH_USLEEP
 extern int tash_usleep(int argc, char **args);
 #endif
+#ifdef CONFIG_PM
+#define TASH_PM_DOMAIN_NAME "TASH"
+#define PM_DRIVER "/dev/pm"
```

Comment:
Is this ("/dev/pm") not exposed in the <tinyara/pm/pm.h>?

Reply:
No

Reply:
pm.c has `#define PM_DRVPATH  "/dev/pm". I think @r-prabu said you should use the same define for the same driver.
There are two files which separately defines `"/dev/pm"` already. It causes mismatch or missing when we modify the driver name. Let's define it at some pm file and use it at every file.

Reply:
Shall I add it to `<tinyara/pm/pm.h>` ?

Reply:
I think yes. You can decide the location.

Reply:
Done